modules/op-using-custom-scc-to-improve-prod-security.adoc

```diff
@@ -0,0 +1,24 @@
+:_content-type: PROCEDURE
+
+[id='op-using-custom-scc-to-improve-prod-security_{context}']
+= Using a custom SCC to improve pod security
+
+Improve the security of your builds and runs by using custom security context constraints (SCCs).
+
+. Procedure
+
+* Edit the `pipelines-scc` and modify the `runAsUser` and `seLinuxContext`.
+
+[source,yaml,subs="attributes+"]
```

Comment:
```suggestion
* Edit the `pipelines-scc` and modify the `runAsUser` and `seLinuxContext`:
+
[source,yaml,subs="attributes+"]
```

see previous comments also about literals like `runAsUser` and `seLinuxContext`, make sure these are written explaining what they are, maybe something like "`runAsUser` and `seLinuxContext` resources" etc